tox.ini

```
@@ -11,7 +11,7 @@ deps =
     django32: -r requirements/django.txt
     django42: -r requirements/django42.txt
```

We need to use default django.txt now
```suggestion
    django42: -r requirements/django.txt
```

---

@UsamaSadiq done.